llama-index-core/llama_index/core/vector_stores/types.py

```diff
@@ -75,6 +75,7 @@ class FilterOperator(str, Enum):
     ANY = "any"  # Contains any (array of strings)
     ALL = "all"  # Contains all (array of strings)
     TEXT_MATCH = "text_match"  # full text match (allows you to search for a specific substring, token or phrase within the text field)
+    ILIKE = "ilike"  # full text match (case insensitive)
```

Comment:
Not sure I like this naming, maybe `TEXT_MATCH_INSENSITIVE` is more consistent?

Reply:
Yeah that's consistent but its kinda lengthy for an operator name.

Reply:
I'll change it to `TEXT_MATCH_INSENSITIVE` right away.

Reply:
Or should I try changing `TEXT_MATCH` to `LIKE`  @logan-markewich ? What do you think?

Reply:
Yea its long, but tbh its more in-line. `LIKE`/`ILIKE` is very SQL-esque